.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -11,12 +11,19 @@ assignees: ''
 For Windows PowerShell 5.1 issues, suggestions, or feature requests please use the following link instead:
 Windows PowerShell [UserVoice](https://windowsserver.uservoice.com/forums/301869-powershell)
 
-This repository is **ONLY** for PowerShell Core 6 and PowerShell 7+ issues.
+This repository is **ONLY** for PowerShell Core 6 and PowerShell 7+ issues:
 
-- Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
-- Search the existing issues.
-- Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
-- Refer to the [known issues](https://docs.microsoft.com/powershell/scripting/whats-new/known-issues-ps6).
+  - If you're seeing behavior that deviates from the intended, documented behavior, report it here, assuming that you can reproduce it in the [latest released version](https://github.com/PowerShell/PowerShell/releases) and that it hasn't already been reported (please search the existing issues).
+  
+  - If you need to get clarity on what the intended behavior is or if you have questions about PowerShell in general, please instead seek help from the community via the following forums:
+  
+    - The [Slack](https://join.slack.com/t/powershell/shared_invite/enQtMzA3MDcxNTM5MTkxLTBmMWIyNzhkYzVjNGRiOTgxZmFlN2E0ZmVmOWU5NDczNTY2NDFhZjFlZTM1MTZiMWIzZDcwMGYzNjQ3YTRkNWM) and [Discord](https://discordapp.com/invite/AtzXnJM) community chat forums (they seamless talk to each other).
+ 
+    - Q&A site [StackOverflow.com](https://stackoverflow.com/questions/tagged/powershell) and the [PowerShell.org](https://powershell.org/forums/) forum for searching or posting questions about PowerShell.
+
+See also:
+- [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
+- [Known Issues](https://docs.microsoft.com/powershell/scripting/whats-new/known-issues-ps6).
```

Comment:
I wonder if it makes sense for us to add links like this into a new section of the README or another doc instead.

I don't have a strong preference -- I suspect it's unlikely that people will read either place. I just haven't found that making our issue templates larger fixes the problems we see with reports, and the README at least renders the markdown nicely.

Then we can just link into the README (or other doc) from the template (although again, lack of markdown rendering means users are probably less likely to follow the link).

Reply:
Thanks, @rjmholt. 

I can see the problem with lack of rendering and therefore lack of clickable links, but, as you state, we have a bootstrapping problem there: the link to a separate README would itself not be clickable.

Either way, it probably makes sense to remove the Markdown formatting altogether from the template.

As for whether a separate README is called for: 

While I can see the problem with having too much text in the template itself, the flip side is that an additional level of indirection - having to follow the README link manually - may make users even less inclined to read the information.

Reply:
Per https://github.com/PowerShell/PowerShell/issues/14474#issuecomment-749265001, it sounds like adding a link to GitHub Discussion may soon be called for as well.